cmd/bucket-replication.go

```diff
@@ -300,12 +300,12 @@ var (
 	// but eventually can be tuned in future,
 	// take only half the CPUs for replication
 	// conservatively.
-	globalReplicationConcurrent = runtime.GOMAXPROCS(0) / 2
+	globalReplicationConcurrent = runtime.GOMAXPROCS(0)
 )
 
 func newReplicationState() *replicationState {
 	rs := &replicationState{
-		replicaCh: make(chan ObjectInfo, globalReplicationConcurrent*2),
```

Comment:
This is on purpose to hold more entries in channel buffer - such that sudden bursts are not blocked.